sql/core/src/main/scala/org/apache/spark/sql/internal/HiveSerDe.scala

```diff
@@ -42,8 +41,7 @@ object HiveSerDe {
         HiveSerDe(
           inputFormat = Option("org.apache.hadoop.hive.ql.io.RCFileInputFormat"),
           outputFormat = Option("org.apache.hadoop.hive.ql.io.RCFileOutputFormat"),
-          serde = Option(conf.getConfString("hive.default.rcfile.serde",
-            "org.apache.hadoop.hive.serde2.columnar.LazyBinaryColumnarSerDe"))),
+          serde = Option("org.apache.hadoop.hive.serde2.columnar.LazyBinaryColumnarSerDe")),
```

Comment:
Below are two related Hive JIRAs:
 - https://issues.apache.org/jira/browse/HIVE-4475
 - https://issues.apache.org/jira/browse/HIVE-6210

 This change has an external impact. I am not sure whether we can remove this.